src/kv/KeyValueDB.h

```diff
@@ -9,6 +9,7 @@
 #include <map>
 #include <optional>
 #include <string>
+#include <string_view>
```

Comment:
(a note that refers to the commit description)
Maybe add, after the perf data, a word about what it is replaced with?